src/equations/2d/ideal_glm_mhd.jl

```diff
@@ -105,7 +105,7 @@ function initial_conditions_rotor(x, t, equation::IdealGlmMhdEquations2D)
   return prim2cons(SVector(rho, v1, v2, v3, p, B1, B2, B3, psi), equation)
 end
 
-function initial_conditions_mhd_blast(x, t, equation::IdealGlmMhdEquations2D)
+function initial_conditions_mhd_blast_wave(x, t, equation::IdealGlmMhdEquations2D)
```

Comment:
We could also think about using multiple dispatch and calling this function still `initial_conditions_blast_wave` (and change it accordingly everywhere). But it's also okay for me to keep it this way. It's good that you made the name more consistent with the other initial conditions :+1:

Reply:
Yeah I went back and forth if I wanted to "overload" the `initial_conditions_blast_wave` similar to `initial_conditions_convergence_test`. I decided to keep the convention that mhd is included for now, but the elixir file being mhd blast wave is probably enough for the user to know what they are about to compute.

Reply:
Both approaches are fine, I think, although I tend to prefer `initial_conditions_blast_wave` slightly. You can decide which name to choose.

Reply:
That approach is growing on me for how "clean" it is and that the elixir directly tells you this information should be clear to everyone. I will address all these comments later today and/or tomorrow